### DISCUSSION TITLE
Auto-add to project workflow when set as a reviewer

### DISCUSSION THREAD

**Discussion by JoshTumath** [community/community#78817] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

#### User need

My team using a Project board to track PR reviews that other teams building on our monorepo have requested us to do. I want PRs to automatically get added to our Project board when our team are set as reviewers.

#### Request

The filter for the auto-add to project workflow only supports `assignee:`, `is:`, `label:`, `milestone:` and `reason:`. It doesn't support any filters related to reviewers or reviewing teams. Could you add a new filter for reviewer users and teams?

![Screenshot 2023-12-12 at 13 42 54](https://github.com/community/community/assets/1760227/8573cf63-6a5b-44dc-9140-d0a56ad4314d)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by escudero89** [comment]
Interested on the same. Is there any news about it?

**Comment by deganmee** [comment]
Same. Wanting to use team-review-requested filter on the Auto-add to Project workflow.
